app/controllers/students_controller.rb

```diff
@@ -97,7 +97,7 @@ def service
       :estimated_end_date
     ])
 
-    estimated_end_date = params.require(:service).permit(*[:estimated_end_date])
+    estimated_end_date = clean_params["estimated_end_date"]
```

Comment:
I noticed that I am getting some `unpermitted params` error in my server log. I simplify things a bit here.